apps/condo/pages/property/[id]/index.tsx

```diff
@@ -1,20 +1,24 @@
 /** @jsx jsx */
-import React from 'react'
+import React, { useState, useMemo } from 'react'
 import { useIntl } from '@open-condo/next/intl'
-import { Row, Col, Typography, Tag, Space, RowProps } from 'antd'
+import { Row, Col, Typography, Space, RowProps, Image, notification } from 'antd'
```

Comment:
Replacing typography with newer one would be awesome :)

Reply:
But it there are a lot of page components that change their appearance after that. And also a few hacks should be done after that change to save original look. For example, now we have that cards with custom size for text
<img width="754" alt="image" src="https://user-images.githubusercontent.com/25844927/211501153-65767656-5391-43b9-a9a1-afa56301d659.png">

After change antd Typography component to uikit we'll lose that ability